lib/webpacker/dev_server.rb

```diff
@@ -60,6 +60,6 @@ def fetch(key)
     end
 
     def defaults
-      config.send(:defaults)[:dev_server]
+      config.send(:defaults)[:dev_server] || {}
```

Comment:
what about `config.dig(:defaults, :dev_server)`?

Reply:
config is not a hash, but a Webpacker::Configuration which has a 'defaults' private method.

Reply:
oh i see